Makefile

```diff
@@ -172,16 +193,10 @@ manifests-ocp: generate-crd controller-gen kustomize
 		--set olm="${OLM}" \
 		--set autoCreateSecret=false \
 		--set createSecurityContextConstraints="true" \
-		--set operator.image="$(MASTER_IMAGE)" > config/deploy/openshift/openshift-csi.yaml
-	grep -v 'app.kubernetes.io/managed-by' config/deploy/openshift/openshift-csi.yaml > config/deploy/openshift/tmp.yaml
-	grep -v 'helm.sh' config/deploy/openshift/tmp.yaml > config/deploy/openshift/openshift-csi.yaml
-	rm config/deploy/openshift/tmp.yaml
-
-	$(KUSTOMIZE) build config/crd | cat - config/deploy/openshift/openshift.yaml > temp
-	mv temp config/deploy/openshift/openshift.yaml
+		--set operator.image="$(MASTER_IMAGE)" > "$(OPENSHIFT_CSIDRIVER_YAML)"
 
-	cat config/deploy/openshift/openshift.yaml > config/deploy/openshift/openshift-olm.yaml
-	cat config/deploy/openshift/openshift.yaml config/deploy/openshift/openshift-csi.yaml > config/deploy/openshift/openshift-all.yaml
+	sed -i "/app.kubernetes.io\/managed-by/d" "$(OPENSHIFT_CSIDRIVER_YAML)" 
+	sed -i "/helm.sh/d" "$(OPENSHIFT_CSIDRIVER_YAML)" 
```

Comment:
sed -i does not work on Mac

Reply:
Please don't flood with same comment every line, this pollutes whole review, thx

Reply:
@meik99 can you try: `brew install gnu-sed` ?
src: https://stackoverflow.com/questions/25124969/sed-replacement-command-not-working-on-mac?msclkid=5bf06597d05011ec875a7ffeb5fb8f85#comment119427508_41029130

Reply:
I don't have a mac

Reply:
I will try 😉

Reply:
> Please don't flood with same comment every line, this pollutes whole review, thx

I do that so occurrences of the same detail are not overlooked and they can be ticked off one by one

Reply:
> > Please don't flood with same comment every line, this pollutes whole review, thx
> 
> I do that so occurrences of the same detail are not overlooked and they can be ticked off one by one

I'd suggest using general comment instead.